src/main/java/de/tum/in/www1/artemis/repository/StudentScoreRepository.java

```diff
@@ -23,8 +23,8 @@ public interface StudentScoreRepository extends JpaRepository<StudentScore, Long
 
     void deleteAllByUser(User user);
 
-    @EntityGraph(type = LOAD, attributePaths = { "user", "exercise", "lastResult", "lastRatedResult" })
-    Optional<StudentScore> findStudentScoreByExerciseAndUser(Exercise exercise, User user);
+    @EntityGraph(type = LOAD, attributePaths = { "user", "exercise" })
+    Optional<StudentScore> findByExercise_IdAndUser_Id(Long exerciseId, Long userId);
```

Comment:
What's the rationale between this naming?

Reply:
Those are Spring JPA Data methods, you can read more about them here: https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.details